tests/api/utils/cancellable_promise.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { CancellablePromise } from "@dataform/api/utils/cancellable_promise";
+import { expect } from "chai";
+
+describe("@dataform/api/utils/cancellable_promise", () => {
+  it("cancel_is_called", () => {
```

Comment:
these don't need underscores, the idea is that they form some meaningful phrase when all glued together